mne/viz/_3d.py

```diff
@@ -1911,6 +1907,22 @@ def _plot_stc(stc, subject, surface, hemi, colormap, time_label,
     with warnings.catch_warnings(record=True):  # traits warnings
         brain = Brain(**kwargs)
     del kwargs
+
+    if using_mayavi:
```

Comment:
May I suggest a different patch for `PySurfer`?

```suggestion
    if using_mayavi:
        # Here we patch to avoid segfault:
        # https://github.com/mne-tools/mne-python/pull/8828
        close_func = brain._close

        def patched_close(*args, **kwargs):
            kwargs["force_render"] = False
            close_func(*args, **kwargs)

        brain._close = patched_close
```

Reply:
In theory, this avoids the call to `_force_render` which calls `process_events`

Reply:
Or maybe patch the public function `close` which is just two lines:

https://github.com/nipy/PySurfer/blame/master/surfer/viz.py#L2298-L2300

```
def close(self):
    self._close()
```

to be
```
def close(self):
    self._close(False)
```

Either way is fine by me, and it seems justified based on this comment

https://github.com/nipy/PySurfer/blame/master/surfer/viz.py#L2324-L2325